File: tool/uapicodegen/uapicodegen/uapi/internal/binary/EncodeBody.py
from typing import TYPE_CHECKING, cast

if TYPE_CHECKING:
    from ...internal.binary.BinaryEncoding import BinaryEncoding


def encode_body(message_body: dict[str, object], binary_encoding: 'BinaryEncoding') -> dict[object, object]:
    from ...internal.binary.EncodeKeys import encode_keys

    return cast(dict[object, object], encode_keys(message_body, binary_encoding))

File: lib/java/src/main/java/io/github/telepact/internal/binary/EncodeBody.java
//|
//|  Copyright The Telepact Authors
//|  SPDX-License-Identifier: Apache-2.0
//|

package io.github.telepact.internal.binary;

import static io.github.telepact.internal.binary.EncodeKeys.encodeKeys;

import java.util.Map;

public class EncodeBody {
    static Map<Object, Object> encodeBody(Map<String, Object> messageBody, BinaryEncoding binaryEncoder) {
        return (Map<Object, Object>) encodeKeys(messageBody, binaryEncoder);
    }
}
